package.json

```diff
@@ -3,20 +3,19 @@
   "version": "1.0.2",
   "description": "",
   "main": "index.js",
-  "scripts": {
-    "test": "scripts/test",
-    "production-test": "scripts/production-test",
-    "dev": "scripts/dev",
-    "up": "scripts/up",
-    "down": "scripts/down",
-    "lint": "scripts/lint",
-    "production-lint": "scripts/production-lint",
-    "start": "scripts/production-start"
-  },
+	"scripts": {
+		"test": "scripts/test",
+		"production-test": "scripts/production-test",
+		"dev": "scripts/dev",
+		"up": "scripts/up",
+		"down": "scripts/down",
+		"lint": "scripts/lint",
+		"production-lint": "scripts/production-lint",
+		"start": "node index.js"
+	},
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@databraid/env-util": "1.1.2",
```

Comment:
@Gwillison415 I believe this library is required for production, could you tell me why you have removed it here?